packages/@okta/i18n/src/properties/login.properties

```diff
@@ -810,6 +811,10 @@ oie.email.verify.alternate.showCodeTextField = Enter a code from the email inste
 oie.email.enroll.subtitle = Please check your email and enter the code below.
 oie.email.return.link.expired.title = Verify with your email
 
+## Additional profile information
+oie.profile.additional.title = Additional Profile information
+oie.profile.additional.secondemail.subtitle = Use a second email to <$1>recover your account</$1> in case you become locked out. This email must be verified after setup.
```

Comment:
this will be parsed from the backend in my next PR

Reply:
widget will still need to use it's own key for customization purposes..